RELEASE-NOTES.txt

```diff
@@ -4,6 +4,7 @@
 20.0
 -----
 - [*] Blaze: Now you can select media attached to the current product while creating Blaze ads. You don't have to scroll through all media in your store. [https://github.com/woocommerce/woocommerce-ios/pull/13540]
+- [internal] Fix SSR parsing error for WPORG login. [https://github.com/woocommerce/woocommerce-ios/pull/13579]
```

Comment:
Nit: This is a bug fix for users authenticated without WPCom, we should mark it with `[*]` instead of `internal`.

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/13579/commits/87c8b8d158333b98424a67af41a9060992319d60